### ISSUE TITLE
Policy cart shows incorrect snapshot change when reform date = snapshot date

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine#349]
![image](https://user-images.githubusercontent.com/6076111/148888731-cdf84d75-abdf-4bb6-b028-65663b5beb3b.png)

**Comment by nikhilwoodruff** [comment]
This might be more of a design bug than a usage bug... It works as expected for me. If I set both to the same date (pressing the button for "Set policy"), then nothing appears in the policy bar. Should we change this?

**Comment by MaxGhenis** [comment]
Yeah hmm I'm not sure how I got this result. It might have happened from clearing it in some order, which would be fixed by #351.

I did just get the policy snapshot to show up though, not sure how but seems like it's inconsistent.
![image](https://user-images.githubusercontent.com/6076111/148979061-4f2745fa-9e52-496b-a720-6723ccd646ca.png)
